packages/url-state-provider/src/index.ts

```diff
@@ -122,17 +135,17 @@ function currentState(stateID) {
  * @param {JSON} state
  * @returns new url string
  */
-function updateState(stateID, state, options = {}) {
+function updateState(stateID: string, state: object, options: UpdateStateOptions = {}) {
   // get current state from URL
-  const newState = URLToState()
+  const newState: { [key: string]: any } = URLToState()
```

Comment:
see my last comment

Reply:
![Screenshot 2024-11-05 at 11 36 24](https://github.com/user-attachments/assets/7578956e-9816-47c3-9652-7804b1efc9ed)

Reply:
Do you know - why